### ISSUE TITLE
False positive format string error with single specifier

### ISSUE THREAD

**Issue by rakyi** [borkdude/clj-kondo#949]
**version**

clj-kondo v2020.07.29

**platform**

macOS 10.15.6 with native clj-kondo

**editor**

Emacs

**problem**

clj-kondo reports false positive format string error.

**repro**

```clojure
(ns repro)

(format "Usage: `%1$s action [arg*]`

Examples:
`%1$s foo
`%1$s bar abc
`%1$s baz`" 'cmd)
```

```bash
clj-kondo --lint repro.clj
repro.clj:3:9: error: Format string expects 3 arguments instead of 1.
linting took 11ms, errors: 1, warnings: 0
```

Note: There are 4 identical format string specifiers in the format string and 1 argument.

**expected behavior**

clj-konto reports no error.

**Comment by borkdude** [comment]
Thanks. Will fix for the next release. Meanwhile you can disable this linter with `{:linters {:format {:level :off}}}` either globally or within the namespace as metadata (`(ns foo {:clj-kondo/config '{...}})`).

**Comment by borkdude** [comment]
Fixed with 365ac5f621d523596354cb9392162af380ee8fe0

**Comment by borkdude** [comment]
FYI, the problem was with a combination of regexes and newlines.

**Comment by borkdude** [comment]
Here is a binary you could use meanwhile: https://10493-176829714-gh.circle-artifacts.com/0/release/clj-kondo-2020.07.30-SNAPSHOT-macos-amd64.zip

**Comment by rakyi** [comment]
Thanks for the quick fix and all the great work you do!
